integration_tests/src/main/python/conditionals_test.py

```diff
@@ -230,6 +231,7 @@ def test_conditional_with_side_effects_case_when(data_gen):
                 conf = test_conf)
 
 @pytest.mark.parametrize('data_gen', [mk_str_gen('[a-z]{0,3}')], ids=idfn)
+@disable_timezone_test
```

Comment:
Can we have a follow on issue to enable/test sequence for all time zones? We don't support timestamps for sequence currently and there are a lot of tests that are failing/skipped for no good reason.